routers/likes.py

```
@@ -0,0 +1,165 @@
+from datetime import datetime
+from typing import Optional
+
+from fastapi import APIRouter, Depends, HTTPException, status, Query
+
+from schemas.like import LikeOut
+from schemas.post import PostLikedOut
+from utils.auth import get_current_user
+from utils.data import read_json, write_json, next_id
+
+from datetime import datetime
```

Ruff (0.14.13)

Redefinition of unused `datetime` from line 1: `datetime` redefined here

Remove definition: `datetime`

(F811)